react/utils/session.tsx

```diff
@@ -26,7 +26,7 @@ const fetchWithRetry = (url: string, init: RequestInit, maxRetries: number = 3):
         return delay(ms)
           .then(() => callFetch(++attempt))
       }
-    })
+    }).catch((error) => console.error(error))
```

Comment:
A better approach would be:

- throw an error on the `then` above if there is an error
- move the retry code to `catch`